src/main/riddl/Members/members.riddl

```diff
@@ -127,6 +127,60 @@ context MembersContext  is {
             }
         }
 
+        state Draft is {
+          fields {
+            info: MembersContext.Info,
+            meta: MembersContext.MetaInfo
+          }
+
+          handler DraftMemberHandler {
+            on command RegisterMember {then error "Member has already been registered."}
+            on command UpdateMemberStatus {
+              example DraftMemberToActiveMember is {
+                given "entity is transitioning from Draft to Active and required fields are non-empty"
+                then "set entity Status to Status field from message"
+                and "set entity lastModifiedBy to actingMember from message"
+                and "set entity lastModifiedOn to today()"
+                and "yield event MemberStatusUpdated"
+              }
+              example DraftMemberToInactiveMember is {
```

Comment:
I think the only state transitions for Member from the Draft state are Draft->Active and Draft->Terminated. I think all others should throw an error.